src/navigation/machines/oid4vciStateNavigation.tsx

```diff
@@ -294,6 +298,16 @@ const navigateReviewCredentials = async (args: OID4VCIMachineNavigationArgs): Pr
       }
     }
 
+    const issuer: Party | undefined = getCredentialIssuerContact(credentialsToAccept[0].uniformVerifiableCredential as VerifiableCredential);
```

Comment:
Is it certain at this point that credentialsToAccept[0] exists?
Also better to extract a const firstCredentialToAccept = credentialsToAccept[0]
There are many instances below of credentialsToAccept[0]

Reply:
yes we are sure, else it will not be here